episodes/15-explanation.md

```diff
@@ -105,8 +105,17 @@ content, approach cutting down the lesson by identifying which learning objectiv
 remove. Then take out the objective(s) and the corresponding assessment(s) and explanatory
 content. You may consider making a [concept map][mental-map-instructor-training] to help identify which objectives depend on one another.
 
-Alternatively, if you decide to keep certain objectives in the lesson, you can add suggestions on which objectives can be skipped to the Instructor Notes for the lesson.
-Remember, reducing the number of lesson objectives can help with managing learner (and instructor) cognitive load.
+Alternatively, if you decide to keep certain objectives in the lesson, you can add 
+suggestions on which objectives can be skipped to the Instructor Notes for the lesson.
+The Instructor Notes are added to the `instructors/` folder of a lesson.
+They are pieces of information that are useful for someone prepping to teach or 
+teaching a specific lesson that self-directed learners might not be 
+interested in reading.
+Instructors can see the notes within the lesson and the instructor notes tab by 
+changing the view in the upper right-hand corner of the lesson to "Instructor View".
```

Comment:
The Workbench also provides the `instructor` fenced div, which can be used to add Instructor Notes "inline" i.e. next to the part of the lesson they are most relevant to.

On a separate point, I agree that this should be put into a callout.

Reply:
Ooo thanks!  I wasn't sure how that worked.

I'm also wondering if we should add this to the sandpaper docs and link it here instead?

Reply:
Ooo might be fun to add an instructor fenced div in the lesson that can be shown here as an example.

Reply:
The instructor note div is described in https://carpentries.github.io/sandpaper-docs/episodes.html#instructor-notes (though you can only see it in https://carpentries.github.io/sandpaper-docs/instructor/episodes.html#instructor-notes, as per instructions).

Reply:
Ok move to call out, linked the transition guide docs, and added an instructor note for them to find.  Was so tempted to rickroll or a photo of a cat in there but kept it simple.

Reply:
> Was so tempted to rickroll or a photo of a cat in there

The late 00's called and they want to know [if I can has cheeseburger](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

Reply:
Great stuff, thank you @sstevens2 & @zkamvar